File: tests/Contexts/Rutine/Exercises/domain/ExerciseNameMother.ts
import { ExerciseName } from '../../../../../src/Contexts/Rutine/Exercises/domain/ExerciseName';
import { WordMother } from '../../../Shared/domain/WordMother';

export class ExerciseNameMother {
  static create(value: string): ExerciseName {
    return new ExerciseName(value);
  }

  static random(): ExerciseName {
    return this.create(WordMother.random());
  }
}

File: src/test/java/com/kuro/rutineweb/exercises/entities/ExerciseNameMother.java
package com.kuro.rutineweb.exercises.entities;

import com.kuro.rutineweb.Shared.entities.WordMother;

public class ExerciseNameMother {
    public static ExerciseName create(String value) {
        return new ExerciseName(value);
    }

    public static ExerciseName random() {
        return new ExerciseName(WordMother.random(50));
    }
}
